application/tests/codeception/common/templates/fixtures/user.php

```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * @var $faker \Faker\Generator
- * @var $index integer
- */
+     * @var $faker \Faker\Generator
```

Comment:
I would suggest setting these Codeception files' comments back to how they were. We apparently have something in our scrutinizer config that encourages Scrutinizer to mess up the indentation here.